radixdlt-java/build.gradle

```diff
@@ -30,9 +30,8 @@ apply plugin: 'com.adarshr.test-logger'
 
 repositories {
     jcenter()
-    maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
     maven { url 'https://jitpack.io' }
-}
+}   
```

Comment:
Arggh.  Looks like you have some trailing whitespace.

Reply:
@clstrfsck whats up with my IDE and whitespace :D I fixed it now!